zip_archive.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
+import os
+import zipfile
+
+# input the dir of folder/file
+dirPath = '/home/yanuarakhid/myfolder'
+
+# input the name
+Zipname = 'myfolder.zip'
```

Comment:
Prefer having this as an argument to the script as well

Reply:
oke i will update soon, thanks for your suggestion